openedx_webhooks/info.py

```diff
@@ -48,6 +48,18 @@ def get_orgs(key):
     orgs = get_orgs_file()
     return set(o for o, info in orgs.items() if info.get(key, False))
 
+def get_person_time_of_pull_request(person, created_at):
+    """
+    Return person data structure as of time of pull request
+    """
+    for before_date in sorted(person.get("before", {})):
+        if created_at.date() <= before_date:
+            before_person = person.get("before")[before_date]
```

Comment:
`.get` here can just be a regular key lookup, since we know that there's a "before" key.